google-cloud-compute-v1/acceptance/google/cloud/compute/v1/query_params_smoke_test.rb

```diff
@@ -0,0 +1,113 @@
+    skip "COMPUTE_TEST_PROJECT must be set before running this test" if @default_project.nil?
+  end
+
+  # Smoke-test the following behaviours: 
+  #   1) set body field to nil
+  #   2) set query param to nil
+  def test_query_params
+    template_name = "rbgapic#{rand 10_000}"
+    igm_name = "rbgapic#{rand 10_000}"
+    template_resource = {
+      name: template_name,
+      properties: {
+        disks: [
+          {
+            initialize_params:
+              {
+                source_image: @image_name
+              },
+            boot: true,
+            auto_delete: true,
+            type: "PERSISTENT"
+          }
+        ],
+      machine_type: "n1-standard-1",
+      network_interfaces: [{ access_configs: [{ name: "default", type: "ONE_TO_ONE_NAT" }] }]
+      }
+    }
+
+    begin
+      puts "inserting instance template #{template_name}"
+      gapic_op = @templates_client.insert project: @default_project, instance_template_resource: template_resource
+      wait_for_global_op gapic_op, "insert instance template #{template_name}"
+
+      igm_resource = {
+        base_instance_name: "rbgapicinst",
+        target_size: 0,
+        instance_template: gapic_op.operation.target_link,
+        name: igm_name
+      }
+
+      puts "inserting instance_group_manager #{igm_resource[:name]}"
+      op = @igm_client.insert project: @default_project, zone: @default_zone, instance_group_manager_resource: igm_resource
+      wait_for_zonal_op op, "insert instance_group_manager #{igm_resource[:name]}"
+
+      igm = @igm_client.get project: @default_project, zone: @default_zone, instance_group_manager: igm_name
+      assert_equal igm.target_size, 0
+
+      resize_op = @igm_client.resize project: @default_project, zone: @default_zone, instance_group_manager: igm_name,
+                                    size: 1
+      wait_for_zonal_op resize_op, "resize"
+
+      igm = @igm_client.get project: @default_project, zone: @default_zone, instance_group_manager: igm_name
+      assert_equal igm.target_size, 1
+
+      resize_op = @igm_client.resize project: @default_project, zone: @default_zone, instance_group_manager: igm_name,
+                                    size: 0
+      wait_for_zonal_op resize_op, "resize"
+
+      igm = @igm_client.get project: @default_project, zone: @default_zone, instance_group_manager: igm_name
+      assert_equal igm.target_size, 0
+    ensure
+      puts "deleting instance_group_manager #{igm_name}"
+      del_op = @igm_client.delete project: @default_project, zone: @default_zone, instance_group_manager: igm_name
+      wait_for_zonal_op del_op, "delete instance_group_manager #{igm_name}"
+
+      @templates_client.delete project: @default_project, instance_template: template_name
+    end
+  end
+
+
+  private
+
+
+  def wait_for_global_op gapic_op, op_type
+    operation = gapic_op.operation
+    $stdout.puts "Waiting for global #{op_type} operation #{operation.name}."
```

Comment:
nit: any reason you have `$stdout.` explicitly for the `puts` lines here but not in `test_query_params`?